scripts/fuzz_shell.js

```diff
@@ -175,7 +175,15 @@ var imports = {
 imports = Asyncify.instrumentImports(imports);
 
 // Create the wasm.
-var instance = new WebAssembly.Instance(new WebAssembly.Module(binary), imports);
+var module = new WebAssembly.Module(binary);
+
+var instance;
+try {
+  instance = new WebAssembly.Instance(module, imports);
+} catch (e) {
+  console.log('Failed to instantiate module');
```

Comment:
```suggestion
  console.log('exception: failed to instantiate module');
```
(the fuzzer script looks for the text "exception")